apps/entropy-tester/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "nodenext",
+    "declaration": true,
+    "rootDir": "src/",
+    "outDir": "./lib",
+    "strict": true,
+    "esModuleInterop": true,
+    "resolveJsonModule": true
+  },
+  "include": ["src"],
+  "exclude": ["node_modules", "**/__tests__/*"]
+}
```

Comment:
I suggest using [my canned tsconfig](https://github.com/cprussin/cprussin-monorepo/blob/main/packages/tsconfig/src/base.json) as a base here, as that will enable a lot of much stricter configs which make the type system much more reliable / trustworthy.  You can use it [like this](https://github.com/pyth-network/pyth-crosschain/blob/main/apps/entropy-explorer/tsconfig.json) if you decide to do so